src/routes/Incidents.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import axios from 'axios'
+import {Table} from './incidents/table';
+
+
+function Incidents() {
+    const [data, setData] = React.useState([]);
+    const url = 'https://contame.herokuapp.com/contame/report';
+    axios.get(url).then((response) => {setData(response.data)});
+
+    return (
+        Table(data)
```

Comment:
Esto anda? Debería ser JSX
```suggestion
        <Table data=data />
```

Y en el otro archivo deberias recibir props, te dejo otro comment mostrandote